### ISSUE TITLE
FAIL: test_WebMercator (test.unitTestSuite.TestSuite)

### ISSUE THREAD

**Issue by cclauss** [mrJean1/PyGeodesy#19]
* Python 2.7 on Linux https://travis-ci.com/cclauss/PyGeodesy/jobs/132401482#L502
* Python 3.7 on Linux https://travis-ci.com/cclauss/PyGeodesy/jobs/132401483#L505
```
======================================================================
FAIL: test_WebMercator (test.unitTestSuite.TestSuite)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/cclauss/PyGeodesy/test/unitTestSuite.py", line 97, in test_WebMercator
    self._run('testWebMercator')
  File "/home/travis/build/cclauss/PyGeodesy/test/unitTestSuite.py", line 31, in _run
    self.assertEqual(x, 0)
AssertionError: 1 != 0
----------------------------------------------------------------------
```

**Comment by mrJean1** [comment]
Which Python is this?  32- or 64-bit?

**Comment by mrJean1** [comment]
Saw that.  Which Python release? 32- or 64-bit? Which Linux distro?

**Comment by mrJean1** [comment]
How/where can I run the failing test, in an attempt to duplicate the problem?

**Comment by cclauss** [comment]
This was all found via #18 so you would have to go to  https://www.travis-ci.com/mrJean1 (login via GitHub credentials) and flit in the repo switch and then merge #18

**Comment by cclauss** [comment]
Ubuntu 16.04.4 LTS (Xenial Xerus) 64bit http://releases.ubuntu.com/16.04/ 

$ __python -c "import platform ; print(platform.architecture(), platform.platform(), platform.uname())"__
```
('64bit', 'ELF') Linux-4.13.0-1017-gcp-x86_64-with-debian-stretch-sid uname_result(system='Linux', node='travis-job-da13418d-37d8-4d32-b6e1-eef5a535b322', release='4.13.0-1017-gcp', version='#21-Ubuntu SMP Thu May 17 14:45:07 UTC 2018', machine='x86_64', processor='x86_64')
```

**Comment by cclauss** [comment]
Ubuntu 14.04.5 LTS (Trusty Tahr) 64bit http://releases.ubuntu.com/14.04/ delivers the same result.

**Comment by mrJean1** [comment]
Thank you.

**Comment by mrJean1** [comment]
I am unable to duplicate the testWebMercator failure (with recent 64-bit Python 2 and 3 versions) and can not log into Travis-CI.com.  Will keep this issue open for the time being.

**Comment by cclauss** [comment]
Does https://www.travis-ci.org/mrJean1 (org, not com) work any better?

**Comment by mrJean1** [comment]
Doesn't work either.  Please let me know (1) when all this works, (2) repeats the testWebMercator error and (3) shows how/where that error can be duplicated.  Errors which can not be duplicated and investigated are not fixable.

**Comment by mrJean1** [comment]
Sir, I appreciate all your effort and the details you finally show.  With respect to Travis, Circle, AppVeyor, CI in general, there hasn’t been any need for that so far and I am not planning to go down that path, yet.

**Comment by mrJean1** [comment]
Fixed in PyGeodesy 18.7.3.
